### ISSUE TITLE
Screen Reader: Sign in: The social network links in the footer are not focused and announced

### ISSUE THREAD

**Issue by olenamihno** [Expensify/App#77555]
## Action Performed:
Prerequisites:
- the user is not signed in

1. Using Android TalkBack or iOS VoiceOver, open the Expensify app
2. Swipe and try to focus the social network links in the footer area.

## Expected Result:
Each social network link should be focused announced with its role and name, e.g.:
- 'Follow us on Facebook, link'

## Actual Result:
The social network links are not focused and announced.

## Area issue was found in:
Sign in

## Failed WCAG checkpoints
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User impact:
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested resolution:
Ensure each social network link is focusable with a screen reader.
https://reactnative.dev/docs/accessibility#accessible

Implement the controls as links.
https://reactnative.dev/docs/accessibility#accessibilityrole

Ensure each link has a descriptive accessibilityLabel, e.g.:
https://reactnative.dev/docs/accessibility#accessibilitylabel

## Environment:
Samsung Galaxy S24 Ultra - Android 14, Partner - Native (No mobile browser)
Apple iPhone 11 - iOS 17.5.1, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos

![Image](https://github.com/user-attachments/assets/ac2c7e78-d921-4fbd-b024-9b9933c77831)

https://github.com/user-attachments/assets/2dce46f8-9c0f-4445-9a7c-350e2cdec037

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** No
**Logs:** https://stackoverflow.com/c/expensify/questions/4856  NA
**Notes/Photos/Videos:** Any additional supporting documentation  Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7005021
**Issue reported by:** NA
**Slack conversation:** NA
